vmb/matrimony/models/profiles.py

```diff
@@ -175,6 +177,30 @@ def __str__(self):
     class Meta:
         db_table = "matrimony_profiles"
 
+    def send_batch_matches_email(self):
+        body = self.get_batch_matches_email_body()
```

Comment:
Check if a `EmailMessage` with `status="DMD"` exists for the current date? If yes, return. Else continue with processing.